packages/api-request/package.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "@wordpress/api-request",
+	"version": "1.0.0",
+	"description": "Utility to call WordPress REST APIs",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"rest-api",
+		"fetch"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/api-request/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"main": "build/index.js",
+	"module": "build-module/index.js",
+	"dependencies": {
+		"jquery": "^3.3.1"
```

Comment:
We use `3.2.1` version in the top level `package.json`. We should align those two or Lerna will install another copy in the package subfolder.